routes/course.js

```diff
@@ -139,6 +139,9 @@ router.post('/api/courses', auth, async (req, res) => {
     await Years.findByIdAndUpdate(retrievedCourse.year_id, {
       $push: { courses: retrievedCourse._id },
     }).exec();
+    await Plans.findByIdAndUpdate(course.plan_id, {
+      updatedAt: Date().toLocaleString(),
```

Comment:
does it make sense to do localestring cuz isn't updateAt of type Date? 
please correct me if i'm wrong, i know nothing about the timestampe feature lol

Reply:
Sorry, I should have elaborated on whats happening there. It doesn't really matter what valued I put in place of Date().toLocaleString(), its pretty much just to tell MongoDB that the object is being updated so it should recalculate the updatedAt field(it does it by itself, not with the value I pass in). So in theory, I could just set updatedAt to an empty string and mongoDB would then set updatedAt to the real current time. Should I add a comment to the code clarifying whats happening there?